src/components/viesti/viestiketju.svelte

```diff
@@ -0,0 +1,89 @@
+<script>
+  import * as R from 'ramda';
+  import * as Formats from '@Utility/formats';
+  import * as Maybe from '@Utility/maybe-utils';
+
+  import * as Viestit from './viesti-util';
+
+  import { currentUserStore } from '@/stores';
```

Comment:
tästä storesta halutaan eroon - anna whoami parametrina tälle

Reply:
lataa whoami yhdessä muiden resurssien kanssa viestit näytöllä

Reply:
katso mallia esim. existing-ketju näytöltä

Reply:
Sivun kontrollerissa ladataan kaikki resurssit ja ne välitetään parametreina eteenpäin muille komponenteille
Jos resurssi halutaan kakuttaa niin käytetään Future.cachea. Storeja ei tarvita tähän.

Reply:
Ach, olikin unohtunut tuo. Pitääkin korjata samointein. Pitäisi yksinkertaistaa koodiakin.